build-system/app-index/url.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+const appendQueryParamsToUrl = (url, params) =>
```

Comment:
Could you use this function: https://github.com/ampproject/amphtml/blob/f89c2990629a8ea95d63561cc150e87e2666c1c3/src/url.js#L222?

Not sure it is the greatest idea to depend on it from build-system though? Probably won't cause any problems.

Reply:
Not possible since they're in different module systems. We could work out a patching/compiling strategy but for this small helper I don't think it's worth it :)